monitoring/defaults_linux.go

```diff
@@ -128,3 +128,12 @@ func DefaultBootConfigParams() health.Checker {
 		},
 	)
 }
+
+// NewDNSChecker sends some default queries to monitor DNS / service discovery health
+func NewDNSChecker() health.Checker {
+	return &DNSChecker{
+		QuestionA: []string{
+			"leader.telekube.local",
```

Comment:
I think this should be a parameter to the `NewDNSChecker` function, satellite should not know anything about our internal DNS names (planet knows it and can pass it here).